src/posit/connect/content/contents.py

```diff
@@ -0,0 +1,287 @@
+from __future__ import annotations
+
+from typing import List, Optional, overload
+
+from requests import Session
+
+from .. import urls
+from ..config import Config
+from ..resources import Resources
+from .content import Content
+
+
+class ContentResources(Resources):
+    """Content resources.
+
+    A resource for many pieces of content.
```

Comment:
It's confusing to me that this class name is a plural, since this is a singular resource, no? 🤔